cvat/apps/documentation/faq.md

```diff
@@ -0,0 +1,72 @@
+# Frequently asked questions
+
+## How to update CVAT
+Before upgrading, please follow the oficcial docker [manual](https://docs.docker.com/storage/volumes/#backup-restore-or-migrate-data-volumes) and backup all CVAT volumes.
+
+To update CVAT, you should clone or download the new version of CVAT and rebuild the CVAT docker images as usual.
+```
+docker-compose build
+```
+and run containers:
+```
+docker-compose up -d
+```
+
+Sometimes the update process takes a lot of time due to a change in the database schema and data. You can check the current status with `docker logs cvat`. Please do not terminate the migration and wait for the process to complete.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[maximum-line-length] Line must be at most 120 characters](https://app.codacy.com/manual/cvat/cvat/pullRequest?prid=5666486)